### PR TITLE
Add note in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/23931

### CODE REVIEW

File: clusters/create_azure.adoc [hunk 1]
```diff
@@ -36,6 +36,8 @@ If you need to create a credential, see link:../credentials/credential_azure.ado
 
 The name of the cluster is used in the hostname of the cluster.
 
+*Note:* The Red Hat Advanced Cluster Managament for Kubernetes controller creates a namespace for clusters you create. Destroying the cluster deletes the namespace. Only use resources of the cluster instance for this namespace.
```
Comment (cadawson) [inline clusters/create_azure.adoc]: This should probably be an important note. There is a chance that they can lose data. 

*Important:* When you create a cluster, the {product-title-short} controller creates a namespace for the cluster and its resources. Ensure that you include only resources for that cluster instance in that namespace. Destroying the cluster deletes the namespace and all of the resources in it.

**Review by cadawson — CHANGES_REQUESTED** [review]
Comment added. Should this also be added to create_bare.adoc? Thanks!

**Comment by oafischer** [comment]
> Comment added. Should this also be added to create_bare.adoc? Thanks!

Good catch on create_bare.adoc @cadawson, that should be included too. I'll add the change there too and address your comment, thanks!

File: clusters/create_azure.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/create_azure.adoc]: I like your version, thanks for providing that. I'll go ahead and implement it everywhere as an important note.

**Review by cadawson — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**